packages/astro/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # astro
 
+## 5.0.0-beta.9
+
+### Patch Changes
+
+- [#12424](https://github.com/withastro/astro/pull/12424) [`4364bff`](https://github.com/withastro/astro/commit/4364bff27332e52f92da72392620a36110daee42) Thanks [@ematipico](https://github.com/ematipico)! - Fixes an issue where an incorrect usage of Astro actions was lost when porting the fix from v4 to v5
+
+- [#12438](https://github.com/withastro/astro/pull/12438) [`c8f877c`](https://github.com/withastro/astro/commit/c8f877cad2d8f1780f70045413872d5b9d32ebed) Thanks [@ascorbic](https://github.com/ascorbic)! - Fixes a bug where legacy content types were generated for content layer collections if they were in the content directory
+
+- [#12377](https://github.com/withastro/astro/pull/12377) [`af867f3`](https://github.com/withastro/astro/commit/af867f3910ecd8fc04a5337f591d84f03192e3fa) Thanks [@ascorbic](https://github.com/ascorbic)! - Adds experimental reponsive image support
```

Comment:
Just checking... is this going to be **the** changeset announcement? Usually this would end up as a minor, and be a bigger deal with a much larger description if it's introducing an experimental flag. (what it is, instructions for enabling the flag, and example of how to use it...)

I know things are happening in little pieces, so maybe this isn't *the one*. Just wanted to catch this in case it is!

Reply:
@sarah11918 https://github.com/withastro/astro/pull/12483